chore: 🔖 release new versions (#1337)
Redocly/redocly-cli: docs/changelog.md

<!-- do-not-remove -->

## 1.5.0 (2023-11-29)

### Minor Changes

- Added new rule `array-parameter-serialization` to require that serialization parameters `style` and `explode` are present on array parameters.

### Patch Changes

> - Deprecated lint-related options in the `bundle` command. The options are going to be removed in the subsequent releases.

Reviewer: ```suggestion
- Deprecated lint-related options in the `bundle` command. The options are marked for removal in a future release.
```
